src/lib/decorators.ts

```diff
@@ -411,16 +419,21 @@ export function eventOptions(options: AddEventListenerOptions) {
  * @category Decorator
  */
 export function queryAssignedNodes(
-    slotName: string = '', flatten: boolean = false) {
+    slotName = '', flatten = false, selector = '') {
```

Comment:
Just note to future selves, we should make decorators take option bags in the future for optional arguments. Too late to change now.